hr_timesheet_predefined_description/tests/test_account_analytic_line.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Tecnativa - Juan José Seguí
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, common
+
+
+class TestAccountAnalyticLine(common.TransactionCase):
```

Comment:
On v14, you should declare it as `common.SavepointCase` and override `setUpClass`, not `setUp`.